src/encoded/static/components/schema.js

```diff
@@ -398,6 +396,16 @@ class DisplayObjectSection extends React.PureComponent {
     render() {
         const { term, schema, schemaName, profilesMap } = this.props;
 
+        // No simple term types in the schema value, so display the schema object
+        const schemaTerm = schema[term];
+        const schemaIsObject = typeof schemaTerm === 'object';
+        const schemaIsArray = Array.isArray(schemaTerm);
+        let simpleTermValuesExist = true;
+        if (schemaIsObject && schemaIsArray) {
```

Comment:
Technically, if schemaIsArray is true, then we know schemaIsObject must also be true.